src/application/api_0_1_0/__init__.py

```diff
@@ -14,3 +14,4 @@
 api.add_resource(routes.UserLogoutResource, '/users/<string:uuid>/updateuserlogin')
 api.add_resource(routes.UserCreateVerificationResource, '/users/<string:uuid>/createverification')
 api.add_resource(routes.UserUpdateVerificationResource, '/users/<string:uuid>/updateverification')
+api.add_resource(routes.UserResource, '/users/<string:uuid>')
```

Comment:
lets change:
`rroutes.UserCreationResource, '/users/registeruser'` to `routes.UserResource, '/users'` you will also need to update the tests that have this endpoint and change them to just `/users`
 `routes.UserResource` to `routes.UserDetailResource`